server/api.py

```diff
@@ -473,6 +473,25 @@ def vaccination(
     return 200, vaccine
 
 
+@api.delete("/vaccination/{player_id}", response={200: None, 400: Response, 500: Response})
```

Comment:
Why use a `delete` method here? Wouldn't it be better to use a `PUT` ? Or even just allowing the POST method to edit, is okay.